### PR TITLE
Pass down create button text props.

### PR DESCRIPTION
**Pull request by niuyunsi**

### CODE REVIEW

File: src/components/NestedForm/NestedFormInput.tsx [hunk 1]
```diff
@@ -7,14 +7,20 @@ import { FormInput } from '../FormInput';
 export interface INestedFormInputState {
 }
 
-export class NestedFormInput extends React.PureComponent<data.IFieldInputProps, INestedFormInputState> {
+export interface INestedFormInputProps {
+    showIndex: boolean;
```
Comment (linweigao) [inline src/components/NestedForm/NestedFormInput.tsx]: make there props optional and give default value in 19.

File: src/components/NestedForm/NestedFormInput.tsx [hunk 2]
```diff
@@ -30,7 +36,7 @@ export class NestedFormInput extends React.PureComponent<data.IFieldInputProps,
         return (
             <div className='form-input-nested'>
                 <div className='form-input-nested-label'>{this.props.field.label}</div>
-                <button type='button' onClick={this.onCreateEntry}>Create</button>
+                {this.renderCreateButton()}
```
Comment (linweigao) [inline src/components/NestedForm/NestedFormInput.tsx]: From our current UI design, the create button is below the all entries, I think it make sense.

Comment (linweigao) [reply src/components/NestedForm/NestedFormInput.tsx]: Move this down to ln40.

File: src/components/NestedForm/NestedFormInput.tsx [hunk 3]
```diff
@@ -85,6 +93,25 @@ export class NestedFormInput extends React.PureComponent<data.IFieldInputProps,
...
+            )
+        }
+
+        if (React.isValidElement(source)) {
+            return React.cloneElement(source, {
+                onClick: this.onCreateEntry,
+                className: classNames(source.props.className, 'form-input-nested-button form-input-nested-create-button'),
+            })
+        }
+    }
```
Comment (linweigao) [inline src/components/NestedForm/NestedFormInput.tsx]: Anyway we should return something, in order not to break the experience. Or throw err is fine.

File: src/components/NestedForm/NestedFormInput.tsx (hunk 1 above)
Comment (niuyunsi) [reply src/components/NestedForm/NestedFormInput.tsx]: Done

File: src/components/NestedForm/NestedFormInput.tsx (hunk 2 above)
Comment (niuyunsi) [reply src/components/NestedForm/NestedFormInput.tsx]: Done

File: src/components/NestedForm/NestedFormInput.tsx (hunk 3 above)
Comment (niuyunsi) [reply src/components/NestedForm/NestedFormInput.tsx]: Done

**Review by linweigao — APPROVED** [review]

**Pull request merged**